lib/inertia_rails/controller.rb

```diff
@@ -41,6 +63,11 @@ def default_render
       end
     end
 
+    def shared_data
+      return {} unless @_inertia_shared_plain_data
```

Comment:
Yes, you're absolutely correct.

Reply:
@buhrmi I think we need more attention in this method. There is this safeguard only for if rendered method is called without a controller. Because in the controller will always setup the initial value

Reply:
I dont think is worth to make something like this
```suggestion
            return {} if @_inertia_shared_plain_data.nil? && @_inertia_shared_blocks.nil?
      return evaluated_blocks if if @_inertia_shared_plain_data.nil?
```

Reply:
I have made a refactor in this one